myft/main.scss

```diff
@@ -71,6 +71,53 @@ $_n-ui-myft-is-silent: nUiHas('myft') == false or $_n-ui-myft-applied == true;
 		}
 	}
 
+	.n-myft-ui--prefer-group {
+		@include oGridRespondTo($from: 'M') {
+			padding-left: $spacing-unit;
+		}
```

Comment:
Unrelated to this, but maybe we should think about, in general, upgrading all our sass warnings to errors? Think the next codebase as a whole is in a better place than it was